### PR TITLE
Convert a few posts to Markdown

### PR DESCRIPTION
**Pull request by XhmikosR**
Fixes a few HTML validation errors along the way.

Also remove the 404 images from /multi-server-continuous-deployment-with-fleet.md.

For npm's posts, I kept the old IDs for backward compatibility.

### CODE REVIEW

**Comment by XhmikosR** [comment]
One thing to note is that the header IDs might differ. There aren't any anchors in those posts though, so one would have to manually find the heading ID and link to it.

Your call how to proceed. We could pass the old IDs manually, but this is simpler :)

File: locale/en/blog/vulnerability/http-server-security-vulnerability-please-upgrade-to-0-6-17.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ category: vulnerability
 slug: http-server-security-vulnerability-please-upgrade-to-0-6-17
 layout: blog-post.hbs
 ---
+
 ## tl;dr
 
 - A carefully crafted attack request can cause the contents of the HTTP parser's buffer to be appended to the attacking request's header, making it appear to come from the attacker.  Since it is generally safe to echo back contents of a request, this can allow an attacker to get an otherwise correctly designed server to divulge information about other requests.  It is theoretically possible that it could enable header-spoofing attacks, though such an attack has not been demonstrated.
 - Versions affected: All versions of the 0.5/0.6 branch prior to 0.6.17, and all versions of the 0.7 branch prior to 0.7.8.  Versions in the 0.4 branch are not affected.
-- Fix: Upgrade to [v0.6.17](http://blog.nodejs.org/2012/05/04/version-0-6-17-stable/, or apply the fix in [c9a231d](https://github.com/joyent/node/commit/c9a231d) to your system.
+- Fix: Upgrade to [v0.6.17](http://blog.nodejs.org/2012/05/04/version-0-6-17-stable/), or apply the fix in [c9a231d](https://github.com/joyent/node/commit/c9a231d) to your system.
```
Comment (XhmikosR) [inline locale/en/blog/vulnerability/http-server-security-vulnerability-please-upgrade-to-0-6-17.md:15]: @nschonni weird this wasn't caught :/

**Comment by XhmikosR** [comment]
Alright, I went ahead and put back the old IDs for backward compatibility.

**Comment by XhmikosR** [comment]
Alright, I believe this is ready for review. We could tweak these further later.

File: locale/en/blog/vulnerability/http-server-security-vulnerability-please-upgrade-to-0-6-17.md (hunk 1 above)
Comment (nschonni) [reply locale/en/blog/vulnerability/http-server-security-vulnerability-please-upgrade-to-0-6-17.md:15]: I think a bunch of the link checking is done through regex. Can't hurt to file an issue with the library though. Not sure if there is a rule that covers links or not

**Comment by XhmikosR** [comment]
If it helps, to review you can check the rich diff for each file on GitHub.

**Comment by XhmikosR** [comment]
Can we get this merged? The plan is to convert most inline HTML to Markdown when possible so that we are consistent.

This also fixes 2 404 errors. There are a few mixed content errors I noticed too, and I'll fix them in another PR.

**Comment by Trott** [comment]
@nodejs/website This needs reviews.

**Review by alexandrtovmach — APPROVED** [review]

**Review by aymen94 — APPROVED** [review]

**Pull request merged**